usr/share/rear/conf/default.conf

```diff
@@ -2169,6 +2168,23 @@ COPY_AS_IS_NBKDC=()
 COPY_AS_IS_EXCLUDE_NBKDC=()
 ####
 
+
+####
+# BACKUP=NFS_SERVER
+##
+# Note: This is for NFS_Server
```

Comment:
Please add hint that this method is for restore only, similar to the other restore only methods.